pkg/logger/logger.go

```diff
@@ -66,12 +69,17 @@ func (l Level) ShouldDisplay(log Level) bool {
 	return l.severity <= log.severity
 }
 
+func (l Level) AsSevereAs(log Level) bool {
+	return l.severity >= log.severity
+}
+
 var (
 	NoneLvl    = Level{id: 0, severity: 0}
 	InfoLvl    = Level{id: 1, severity: 300}
 	VerboseLvl = Level{id: 2, severity: 200}
 	DebugLvl   = Level{id: 3, severity: 100}
 	WarnLvl    = Level{id: 4, severity: 400}
+	ErrorLvl   = Level{id: 5, severity: 500}
```

Comment:
nbd but maybe put these in order (i.e. put Debug at top)?

Reply:
done